doc/nrf/releases/release-notes-latest.rst

```diff
@@ -172,6 +172,10 @@ nRF9160
 
     * Updated the default server URL to ``mqtt.eclipseprojects.io``.
 
+  * :ref:`lte_link_control` library:
```

Comment:
Looks that a proper ref would be `lte_lc_readme`.

Reply:
Nice, I'll give it a try

Reply:
Documentation Build / build (pull_request) Successful in 18m :)